api/src/main/java/org/openmrs/module/fhir2/api/search/param/ConditionSearchParams.java

```diff
@@ -40,10 +40,12 @@ public class ConditionSearchParams extends BaseResourceSearchParams {
 	
 	private DateRangeParam recordedDate;
 	
+	private TokenAndListParam tag;
+	
 	@Builder
 	public ConditionSearchParams(ReferenceAndListParam patientParam, TokenAndListParam code,
 	    TokenAndListParam clinicalStatus, DateRangeParam onsetDate, QuantityAndListParam onsetAge,
-	    DateRangeParam recordedDate, TokenAndListParam id, DateRangeParam lastUpdated, SortSpec sort,
+	    DateRangeParam recordedDate, TokenAndListParam tag, TokenAndListParam id, DateRangeParam lastUpdated, SortSpec sort,
```

Comment:
Here we should allow searching by category rather than tag, which will still be a `TokenAndListParameter`.